metadata/nz_building_outlines.xml

```diff
@@ -133,9 +133,9 @@ Please refer to the [NZ Building Outlines Data Dictionary] (https://nz-buildings
 
 **Related building outlines data**
 
-[NZ Building Outlines Full History] (https://data.linz.govt.nz/layer/xxxxx) - contains all combinations of building outlines from multiple years of imagery that have existed since the beginning of this dataset, and the dates when each building outline existed in the associated aerial imagery.
+[NZ Building Outlines All Sources] (https://data.linz.govt.nz/layer/101292) - contains all combinations of building outlines from multiple years of imagery that have existed since the beginning of this dataset, and the dates when each building outline existed in the associated aerial imagery.
```

Comment:
```suggestion
[NZ Building Outlines (All Sources)] (https://data.linz.govt.nz/layer/101292) - contains all combinations of building outlines from multiple years of imagery that have existed since the beginning of this dataset, and the dates when each building outline existed in the associated aerial imagery.
```